### ISSUE TITLE
docs(omnisite): "GET STARTED" below headline should be more noticeable or be removed

### ISSUE THREAD

**Issue by jnehlmeier** [sveltejs/svelte.dev#513]
### Describe the bug

Currently it is easy to miss the "GET STARTED" link below the headline. The headline text is large and the svelte logo is large and complex and thus distracting. Small red text below the headline looks like a subtitle. 

Styling the link as larger red button would make it more noticeable.

<img width="512" alt="get-started" src="https://github.com/user-attachments/assets/5dadcb04-413f-48cf-b3a1-a91611a90273">

### Reproduction

--

### System Info

```shell
--
```

### Severity

annoyance

**Comment by Rich-Harris** [comment]
I feel differently. For one thing it's a link, not a button, and so it shouldn't look like a button. (I know a lot of sites do this, but they shouldn't!) It could _maybe_ have an arrow-pointing-right icon or something like that to make it more obviously a link. 

But my main disagreement is with this:

> Currently it is easy to miss the "GET STARTED" link below the headline

Really? Because there's two interpretations of this:

1. If you're looking for a 'get started' link it's hard to find. I do not believe this to be the case
2. It's not so in-your-face that it demands your attention whether you're looking for it or not. This is a good thing!

Opened #529 which adds the arrow and makes it a _tiny_ bit more prominent, but not in an annoying distracting way.

**Comment by jnehlmeier** [comment]
> I feel differently. For one thing it's a link, not a button, and so it shouldn't look like a button. (I know a lot of sites do this, but they shouldn't!) It could _maybe_ have an arrow-pointing-right icon or something like that to make it more obviously a link.

Yeah I generally agree with it. It has an href / url so it should be a link. Still it feels a bit lost below the headline. But maybe it is just me, hard do describe. Maybe it is also the positioning itself. We have a large headline, a large logo, a large "/'svelt/" thing and in between a small "Get started" link. It could also go below the description paragraph. At least I have then read what svelte is and then "get started".

At the end it comes down to how prominent you want that get started link. At least I can say on mobile I first didn't noticed it.

There is also a minor issue in consistency because "/'svelt/" is red as well but not a link. So it is not immediately clear that "get started" is a link if you never visited the site before. It is minor because it is a call to action wording but it is inconsistent.





> 
> But my main disagreement is with this:
> 
> > Currently it is easy to miss the "GET STARTED" link below the headline
> 
> Really? Because there's two interpretations of this:
> 
> 1. If you're looking for a 'get started' link it's hard to find. I do not believe this to be the case
> 2. It's not so in-your-face that it demands your attention whether you're looking for it or not. This is a good thing!
> 
> Opened #529 which adds the arrow and makes it a _tiny_ bit more prominent, but not in an annoying distracting way.

**Comment by jnehlmeier** [comment]
Just seen the arrowed version. Already makes a difference. Maybe a little more space above it would make it even clearer. On mobile the CTA could have equal spacing to the headline above and the logo below.

**Comment by Rich-Harris** [comment]
Agree with that — I added a touch more spacing and merged #529, so I'll close this. Thanks!
